nbs/docs/tutorials/ConformalPrediction.ipynb

```diff
@@ -0,0 +1,14030 @@
+{
```

Comment:
<p>Before the references, we should include a future work and an acknowledgements section. Here are some ideas:</p><p> </p><blockquote><strong>Future work </strong></blockquote><blockquote>Conformal prediction has become a powerful framework for uncertainty quantification, providing well-calibrated prediction intervals without making any distributional assumptions. Its use has surged in both academia and industry over the past few years. We'll continue working on it, and future tutorials may include:</blockquote><blockquote><br></blockquote><blockquote>- Exploring larger datasets</blockquote><blockquote>- Incorporating industry-specific examples</blockquote><blockquote>- Investigating specialized methods like the jackknife+ that are closely related to conformal prediction (for details on the jackknife+ see [here](https://valeman.medium.com/jackknife-a-swiss-knife-of-conformal-prediction-for-regression-ce3b56432f4f)).</blockquote><blockquote><br></blockquote><blockquote>If you're interested in any of these, or in any other related topic, please let us know by opening an issue on [GitHub](https://github.com/Nixtla/statsforecast/issues)</blockquote><p> </p><blockquote><strong>Acknowledgements </strong></blockquote><blockquote>We would like to thank [Kevin Kho](https://github.com/kvnkho) for writing this tutorial, and Valeriy [Manokhin](https://github.com/valeman) for his expertise on conformal prediction, as well as for promoting this work.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/597/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Ok will add these!